### PR TITLE
3/4 perspective donut boxes

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
3/4ths donut boxes, courtesy of eris.

## Why It's Good For The Game
Bye bye weird perspective donut boxes.

## Images of changes
image

## Testing
I looked at the donut box

## Changelog
:cl:
tweak: Donut boxes are 3/4th perspective now
/:cl:

### CODE REVIEW

**Comment by Contrabang** [comment]
oh god no,... the soul!!

**Comment by roarbark** [comment]
looks better than what we have

**Comment by Sirryan2002** [comment]
The perspective is better but the actual box looks weird, it doesn't feel like a box and the lid doesn't look like it completely seals the box if closed

**Comment by Hematoma** [comment]
Rather than 3/4 this looks like a front view.

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -50,12 +50,9 @@
 	. = ..()
 	for(var/I = 1 to length(contents))
 		var/obj/item/reagent_containers/food/snacks/donut/donut = contents[I]
-		var/icon/new_donut_icon = icon('icons/obj/food/containers.dmi', "donut_[donut.donut_sprite_type]")
-		new_donut_icon.Shift(EAST, 3 * (I-1))
+		var/icon/new_donut_icon = icon('icons/obj/food/containers.dmi', "[(I - 1)]donut[donut.donut_sprite_type]")
```
Comment (hal9000PR) [inline code/game/objects/items/weapons/storage/fancy.dm:53]: do we really need 6 different donut sprites for each donut type, when they are just translated right? can't you get the shift system to work with this?

**Comment by atteria** [comment]
What's with the weird black outline?

File: code/game/objects/items/weapons/storage/fancy.dm (hunk 1 above)
Comment (S34NW) [reply code/game/objects/items/weapons/storage/fancy.dm:53]: The old system translated, but the new one unfortunately doesn't work that way with how the donuts are shaded

**Comment by Miraviel** [comment]
To me, it looks a bit strange:

1) Yellow: Why is there shading outside the outline?
2) Blue: The very sharp edge looks artificial here.
3) Green: The lid is not wide enough, it does not cover the box.
4) Orange: That looks like a lot of banding, I think less is more here?
5) The outline is very very dark, compared to our washing machines (that are also white-grayish).

image

(first: original, second: issues outlined, third: version I think is cleaner(??))

**Review by Miraviel — CHANGES_REQUESTED** [review]
I forgot to make my comment a review, please look above for the actual review :gatto:

**Comment by S34NW** [comment]
if you pass that sprite over ill slap it on with credit

**Comment by LiberatedWaveMan** [comment]
i'm in favour of changing up sprites and moving to 3/4 stuff, but this doesn't really... feel right. it looks more like a front-view, and it's weirdly outlined, as others mentioned

**Comment by S34NW** [comment]
image
Box cleaned up similar to miraviel's example

**Comment by Sirryan2002** [comment]
This perspective looks much better and I think in it's now current state it would make a good addition 👍

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**